docs/tables/mongodbatlas_cluster.md

```diff
@@ -0,0 +1,67 @@
+# Table: mongodbatlas_cluster
+
+MongoDB Atlas cluster is a NoSQL Database-as-a-Service offering in the public cloud (available in Microsoft Azure, Google Cloud Platform, Amazon Web Services).
+
+## Examples
+
+### Basic Info
```

Comment:
```suggestion
### Basic info
```

`info` should not be capitalized, as per https://steampipe.io/docs/develop/table-docs-standards#basic-info-example